BlazarUI/app/scripts/components/feedback/FeedbackForm.jsx

```diff
@@ -7,16 +7,21 @@ import Alert from 'react-bootstrap/lib/Alert';
 import Feedback from '../../models/Feedback';
 import FeedbackActions from '../../actions/feedbackActions';
 import FeedbackStore from '../../stores/feedbackStore';
+import Cookies from 'js-cookie';
 
 class FeedbackForm extends Component {
 
   constructor() {
     bindAll(this, 'toggleShow', 'submitFeedback', 'handleNameChange', 'handleMessageChange', 'resetForm', 'onStatusChange');
+
+    name = Cookies.get(config['usernameCookie']);
+    name = (name == undefined) ? '' : name;
```

Comment:
Should be able to simplify these two lines to:

`name = Cookies.get(config['usernameCookie']) || '';`